features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/app.jag

```diff
@@ -0,0 +1,58 @@
+<%
+    var log = new Log("Services login app");
+    var url = 'https://localhost:9443/client-registration/v0.14/register';
+    var data = { "callbackUrl": "localhost", "clientName": request.getParameter("application", "UTF-8") , "owner": "admin", "grantType": "password refresh_token", "saasApp": true };
+    var result = post(url, JSON.stringify(data) , {
+        "Authorization": "Basic YWRtaW46YWRtaW4=",
+        "Content-Type": "application/json"
+    }, "json");
+
+    //curl -k -d "grant_type=password&username=admin&password=admin&scope=apim:subscribe" -H "Authorization: Basic SGZFbDFqSlBkZzV0YnRyeGhBd3liTjA1UUdvYTpsNmMwYW9MY1dSM2Z3ZXpIaGM3WG9HT2h0NUFh" https://localhost:8243/token
+
+    var clientId = result.data.clientId;
+    var clientSecret = result.data.clientSecret;
+    var Base64 = org.apache.axiom.om.util.Base64;
+    var String = Packages.java.lang.String;
+    var base64encoded = Base64.encode(new String(clientId+":"+clientSecret).getBytes());
```

Comment:
need to format

Reply:
@isharac , Can you please point out which line needs formatting?

Reply:
I think it's the spacing fixed with this commit https://github.com/wso2/carbon-apimgt/pull/6209/commits/102d37edcc4cdbc76b8d9d391d21e164dbafb19f

Reply:
Yes, it's spacing but it should be like
Base64.encode(new String(clientId + ":" + clientSecret).getBytes());

Reply:
Got it , Fixed with this commit https://github.com/wso2/carbon-apimgt/pull/6209/commits/5a924a6f63d22fca6bcb861b5a1669876a21148a